openc3/spec/interfaces/http_client_interface_spec.rb

```diff
@@ -42,7 +58,225 @@ module OpenC3
         expect(i.connection_string).to eql "http://127.0.0.1:8080"
       end
     end
+=end
+
+    before(:each) do
+      @interface = HttpClientInterface.new('example.com', 8080, 'https', 10, 15, 5, true)
+    end
+
+    describe "#initialize" do
+      it "initializes with default parameters" do
+        interface = HttpClientInterface.new('example.com')
+        expect(interface.instance_variable_get(:@hostname)).to eq('example.com')
+        expect(interface.instance_variable_get(:@port)).to eq(80)
+        expect(interface.instance_variable_get(:@protocol)).to eq('http')
+      end
```

Comment:
Combine with the existing ... check for i.name

Reply:
Done